pkg/config/integration_config.go

```diff
@@ -136,3 +138,24 @@ func NewIntegrationTestConfig(ctx context.Context, tb testing.TB) (*IntegrationT
 
 	return &cfg, db
 }
+
+// E2EConfig represents configurations to run server E2E tests.
+type E2EConfig struct {
+	APIServerURL string           `env:"E2E_APISERVER_URL"`
+	AdminAPIURL  string           `env:"E2E_ADMINAPI_URL"`
+	ProjectID    string           `env:"PROJECT_ID"`
```

Comment:
Can we use `E2E_PROJECT_ID` here? After today's incident, I want to make sure e2e is totally isolated from any other envvars used for dev or prod.

Reply:
Done